ldes-fragmentisers/ldes-pagination/src/main/java/be/vlaanderen/informatievlaanderen/vsds/services/PageCreator.java

```diff
@@ -0,0 +1,65 @@
+package be.vlaanderen.informatievlaanderen.vsds.services;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.entities.LdesFragment;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.repository.LdesFragmentRepository;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.services.NonCriticalTasksExecutor;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragment.valueobjects.TreeRelation;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragmentrequest.valueobjects.FragmentPair;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.tree.relations.TreeRelationsRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static be.vlaanderen.informatievlaanderen.ldes.server.domain.constants.RdfConstants.*;
+import static be.vlaanderen.informatievlaanderen.ldes.server.domain.constants.RdfConstants.PREVIOUS_PAGE_RELATION;
+
+public class PageCreator {
+	private final LdesFragmentRepository ldesFragmentRepository;
+	private final TreeRelationsRepository treeRelationsRepository;
+	private final NonCriticalTasksExecutor nonCriticalTasksExecutor;
+	private static final Logger LOGGER = LoggerFactory.getLogger(PageCreator.class);
+
+	public PageCreator(LdesFragmentRepository ldesFragmentRepository,
+			TreeRelationsRepository treeRelationsRepository,
+			NonCriticalTasksExecutor nonCriticalTasksExecutor) {
+		this.treeRelationsRepository = treeRelationsRepository;
+
+		this.ldesFragmentRepository = ldesFragmentRepository;
+		this.nonCriticalTasksExecutor = nonCriticalTasksExecutor;
+	}
+
+	public LdesFragment createFirstFragment(LdesFragment parentFragment) {
+		return createFragment(parentFragment, "1");
```

Comment:
The ever lasting 0 vs 1 starting element argument.. I'm okay with this, as long as we mention it clearly in the readme that the first page is page 1

Reply:
I have added this in the readme. I have also made the first page number into a constant so it can be easily changed into a 0